public/prelert_swimlane_vis_params.html

```diff
@@ -1,53 +1,63 @@
 <div class="form-group prl-swimlane-vis-options">
   <h4>Band thresholds</h4>
-  <p>Enter the lower threshold for each color band:</p>
+  <p>Enter the lower threshold and color for each band:</p>
 
   <div class="threshold-input-container">
-    <i class="fa fa-circle icon-severity-low" aria-hidden="true"></i>
+    <i class="fa fa-circle icon-severity-low" ng-style="{'color':vis.params.lowThresholdColor}" aria-hidden="true"></i>
     <input type="number" name="low_threshold_input" step="any" class="form-control"
       ng-model="vis.params.lowThreshold" ng-model-options="{ updateOn: 'blur' }"
       less-than="{{vis.params.warningThreshold}}"
       ng-required="true">
-    <kbn-info info="Defines lower threshold for values between {{vis.params.lowThreshold}} and {{vis.params.warningThreshold}}"></kbn-info>
+    <kbn-info info="Defines lower threshold for values between {{vis.params.lowThreshold}} and {{vis.params.warningThreshold}}."></kbn-info>  
```

Comment:
No need for the full stop in the tooltip for the first field since it is only one sentence.